### PR TITLE
[EG-1164] Removes link to templates. Emphasizes license terms. Emphasizes non-p…

### PR DESCRIPTION
**Pull request by joeldudleyr3**
…rod readiness.

### CODE REVIEW

File: content/en/docs/corda-enterprise/4.5/node/deploy/one-click-deployment.md [hunk 1]
```diff
@@ -16,15 +16,17 @@ weight: 65
...
 
-The templates can be found on the Azure Marketplace or downloaded from the [`azure-resource-management-templates` repo](https://github.com/corda/azure-resource-management-templates/) on GitHub.
+{{< note >}}The nodes deployed using these templates are not production-ready. In particular, they use H2 as their database, which is not a supported configuration, as per the [Platform Support Matrix](../../platform-support-matrix.md#node-databases). In addition, they are not configured to use hardware security modules or Corda Enterprise Firewall instances, and are not deployed in high-availability mode. {{< /note >}}
 
-The deployed nodes will use H2 databases, will not be configured to use hardware security modules or Corda Enterprise Firewall instances, and will not be deployed in high-availability mode.
+{{< note >}}CORDA ENTERPRISE may be used for evaluation purposes for 90 days pursuant to the Software Evaluation License Agreement. Any use beyond this (e.g. in production deployments) requires a commercial license. Please contact sales@r3.com for more information.
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.5/node/deploy/one-click-deployment.md:27]: CORDA ENTERPRISE should not be capitalised.
sales@r3.com - probably add as an email link
license agreement needs a link too
e.g. > for example, ...
license > licence

File: content/en/docs/corda-enterprise/4.5/node/deploy/one-click-deployment.md [hunk 2]
```diff
@@ -16,15 +16,17 @@ weight: 65
...
+
+The Software Evaluation License Agreement for this product can be viewed from https://www.r3.com/corda-enterprise-evaluation-license. {{< /note >}}
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.5/node/deploy/one-click-deployment.md:29]: Remove this line, the link should be added above

File: content/en/docs/corda-enterprise/4.5/node/deploy/one-click-deployment.md [hunk 3]
```diff
@@ -16,15 +16,17 @@ weight: 65
...
 # One-click Corda deployment
 
-This document describes how Corda nodes and networks can be deployed in one click in an Azure environment. It uses three Azure deployment templates:
+This document describes how Corda nodes and networks can be deployed in one click in an Azure environment. There are three offerings that can be found on the Azure Marketplace:
 
-* **CENM Deployment**: CENM is deployed onto a newly-created Azure Kubernetes Service cluster.
-* **Nodes Onto An Existing Network**: Deploy Nodes onto an newly-created Azure Kubernetes Service cluster that connects to an externally running network.
-* **Corda One Click**: CEMN is deployed onto a newly-created Azure Kubernetes Service cluster and then nodes running a custom CorDapp are deployed.
+* **CENM Deployment**: A CENM instance is deployed onto a newly-created Azure Kubernetes Service cluster.
+* **Nodes Onto An Existing Network**: Corda Enterprise Nodes are deployed onto an newly-created Azure Kubernetes Service cluster that connects to an externally running network.
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.5/node/deploy/one-click-deployment.md:22]: an newly > a newly

Comment (ivanterziev-r3) [reply content/en/docs/corda-enterprise/4.5/node/deploy/one-click-deployment.md:22]: externally running network - could be phrased better

File: content/en/docs/corda-enterprise/4.5/node/deploy/one-click-deployment.md [hunk 4]
```diff
@@ -16,15 +16,17 @@ weight: 65
 
 # One-click Corda deployment
 
-This document describes how Corda nodes and networks can be deployed in one click in an Azure environment. It uses three Azure deployment templates:
+This document describes how Corda nodes and networks can be deployed in one click in an Azure environment. There are three offerings that can be found on the Azure Marketplace:
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.5/node/deploy/one-click-deployment.md:19]: what is an offering?

**Comment by joeldudleyr3** [comment]
@ivanterziev-r3 Addressed.

FYI, please don't merge until James M/James H have approved.

**Comment by ivanterziev-r3** [comment]
@JamesHR3 @jmacmahonr3  Can you review please?

**Comment by ivanterziev-r3** [comment]
Thanks @JamesHR3 . I'll merge it as is now and if @jmacmahonr3 has anything to add for updating we'll fix in a new PR.

**Pull request merged**